composer/cli/launcher.py

```diff
@@ -313,19 +314,20 @@ def _get_file(format: str):
                         local_world_size=nproc,
                         node_rank=node_rank,
                     )
-                    return open(filename, 'x+')
+                    return open(filename, 'a+')
```

Comment:
curious why you needed to make this change? I think failing if the log file already exists is a reasonable behavior.

Reply:
good q, I don't think we actually need a+ because none of the gpu log files should exist at this point.